util/deephash/deephash.go

```diff
@@ -116,19 +115,25 @@ type appenderTo interface {
 	AppendTo([]byte) []byte
 }
 
-func (h *hasher) uint(i uint64) {
-	binary.BigEndian.PutUint64(h.scratch[:8], i)
-	h.bw.Write(h.scratch[:8])
+func (h *hasher) hashUint8(i uint8) {
+	h.bw.WriteByte(i)
 }
-
-func (h *hasher) int(i int) {
-	binary.BigEndian.PutUint64(h.scratch[:8], uint64(i))
+func (h *hasher) hashUint16(i uint16) {
+	binary.BigEndian.PutUint16(h.scratch[:2], i)
```

Comment:
Nit: Let's use LittleEndian by default in cases in which it doesn't matter. Our most active platforms are all little-endian.

Reply:
Done.